nautobot/dcim/models/device_components.py

```diff
@@ -56,6 +57,7 @@
     "RearPort",
 )
 
+from functools import cached_property
```

Comment:
This is Python 3.8+ only. Please use [`django.utils.functional.cached_property`](https://docs.djangoproject.com/en/4.1/ref/utils/#django.utils.functional.cached_property) instead.